### ISSUE TITLE
Bug while defining FederatedModel

### ISSUE THREAD

**Issue by daviddelriod** [intel/openfl#715]
**Description*
The problem is that I am trying to emmulate the MNIST pytorch tutorial with another dataset but when defining the FederatedModel class, I get an error due to the Net having issues with the format of the data returned by the Federated Dataset.

**TypeError**: conv2d(): argument 'input' (position 1) must be Tensor, not torch.Size

**Expected behavior**
The issue comes when I try to instanciate the class. It seems that the first convolutional layer is receiving a torch.Size objet instead 
of a torch.Tensor. I have checked the data introduced to the Federated Dataset class and is the same type as the one in the MNIST example, as well as the same dimensions.

**Additional Inf**
 - I am working with Visual Studio Code. 
 -  torch version = '1.8.1+cpu'   
 -  python version = 3.7.12
 -  latest version of openfl 

**Code**
  `class Net(nn.Module):
       def __init__(self):
           super(Net, self).__init__()
           self.conv1 = nn.Conv2d(3, 32, kernel_size=3, stride=1, padding=1)
           self.conv2 = nn.Conv2d(32, 64, kernel_size=3, stride=1, padding=1)
           self.conv3 = nn.Conv2d(64, 128, kernel_size=3, stride=1, padding=1)
           self.pool = nn.MaxPool2d(kernel_size=2, stride=2, padding=0)
           self.dropout = nn.Dropout(0.25)`
        
    def forward(self, x, *args):
        print(x)
        x = self.pool(F.relu(self.conv1(x)))
        x = self.pool(F.relu(self.conv2(x)))
        x = self.pool(F.relu(self.conv3(x)))
        print(x.shape)
        x = x.view(-1, 128 * x.shape[2] * x.shape[3])
        x = self.dropout(x)
        self.fc1 = nn.Linear(x.shape[1], 128)
        x = F.relu(self.fc1(x))
        x = self.dropout(x)
        self.fc2 = nn.Linear(128, classes)
        x = self.fc2(x)
        return x`

`fl_data = FederatedDataSet(train_features, np.array(train_labels), val_features, 
          np.array(val_labels), batch_size=32,num_classes=2)`

`fl_model = FederatedModel(build_model = cnn, optimizer = optimizer, data_loader = fl_data)`

`train_features.shape = torch.Size([4172, 3, 150, 150])`
`train_labels.shape = torch.Size([4172])`

![c](https://user-images.githubusercontent.com/64251001/214718320-61c96c6e-3c32-42ce-816c-762fad223580.png)

**Comment by daviddelriod** [comment]
The problem was that i was instantiating the class (cnn = Net() ) and then passing the object to FederatedModel but you just have to pass the Net directly:
`fl_model = FederatedModel(build_model = cnn, optimizer = optimizer, data_loader = fl_data)`
